stroom-search/stroom-search-elastic/src/main/java/stroom/search/elastic/ElasticIndexResourceImpl.java

```diff
@@ -43,16 +43,19 @@ class ElasticIndexResourceImpl implements ElasticIndexResource, FetchWithUuid<El
     private final Provider<ElasticClusterStore> elasticClusterStoreProvider;
     private final Provider<ElasticIndexStore> elasticIndexStoreProvider;
     private final Provider<DocumentResourceHelper> documentResourceHelperProvider;
+    private final ElasticConfig elasticConfig;
 
     @Inject
     ElasticIndexResourceImpl(
             final Provider<ElasticClusterStore> elasticClusterStoreProvider,
             final Provider<ElasticIndexStore> elasticIndexStoreProvider,
-            final Provider<DocumentResourceHelper> documentResourceHelperProvider
+            final Provider<DocumentResourceHelper> documentResourceHelperProvider,
+            final ElasticConfig elasticConfig
```

Comment:
Config objects need to be injected using a Provider<> so that they can be changed at runtime.